package.json

```diff
@@ -54,8 +54,8 @@
     "ethers": "^5.1.0"
   },
   "devDependencies": {
-    "@arbitrum/nitro-contracts": "1.1.0",
-    "@arbitrum/token-bridge-contracts": "1.2.0",
+    "@arbitrum/nitro-contracts": "^1.1.0",
+    "@arbitrum/token-bridge-contracts": "^1.2.0",
```

Comment:
let's bump to ^1.2.1 to get rid of sol2uml
https://github.com/OffchainLabs/token-bridge-contracts/releases/tag/v1.2.1

Reply:
bumped in https://github.com/OffchainLabs/arbitrum-sdk/pull/310/commits/af5586fc54fefc204d86f0eceef671f42ebd8fcf